Makefile

```diff
@@ -280,29 +278,6 @@ errcheck:
 	@echo "Running errcheck..."
 	@errcheck $(GO_PACKAGES)
 
-.PHONY: revive
-revive:
-	@hash revive > /dev/null 2>&1; if [ $$? -ne 0 ]; then \
-		$(GO) install github.com/mgechev/revive@v1.1.2; \
-	fi
-	@revive -config .revive.toml -exclude=./vendor/... ./...
-
-.PHONY: misspell-check
-misspell-check:
-	@hash misspell > /dev/null 2>&1; if [ $$? -ne 0 ]; then \
-		$(GO) install github.com/client9/misspell/cmd/misspell@v0.3.4; \
-	fi
-	@echo "Running misspell-check..."
-	@$(GO) run build/code-batch-process.go misspell -error -i unknwon '{file-list}'
```

Comment:
It seems to me that this shouldn't be touched, as it's using `build/code-batch-process.go` which seems to be special build made to batch this misspell action. Not sure why, maybe one of the other maintainers can shed some light on that.

Reply:
`build/code-batch-process.go` is only a wrapper for CLI.

In Windows, you can not pass more than 32K bytes via CLI arguments, so the`code-batch-process.go` split the file names into batches. We can just ignore the wrapper during refacotoring.

Reply:
ps: the `build/code-batch-process.go` came from https://github.com/go-gitea/gitea/pull/17684 to fix https://github.com/go-gitea/gitea/issues/14438 , so do not worry about it 😊